### PR TITLE
chore: Remove addons that are no longer supported

### PR DESCRIPTION
**Pull request by bryantbiggs**
### What does this PR do?

- Remove addons that are no longer supported

### Motivation

- The removed addons can be provisioned through alternate means, but are no longer included in the blueprints addons

### More

- [ ] Yes, I have tested the PR using my local account setup (Provide any test evidence report under Additional Notes)
- [ ] Yes, I have added a new example under [examples](https://github.com/aws-ia/terraform-aws-eks-blueprints/tree/main/examples) to support my PR
- [ ] Yes, I have created another PR for add-ons under [add-ons](https://github.com/aws-samples/eks-blueprints-add-ons) repo (if applicable)
- [ ] Yes, I have updated the [docs](https://github.com/aws-ia/terraform-aws-eks-blueprints/tree/main/docs) for this feature
- [ ] Yes, I ran `pre-commit run -a` with this PR

**Note**: Not all the PRs require a new example and/or doc page. In general:
- Use an existing example when possible to demonstrate a new addons usage
- A new docs page under `docs/add-ons/*` is required for new a new addon

### For Moderators

- [ ] E2E Test successfully complete before merge?

### CODE REVIEW

File: modules/crossplane/README.md [hunk 1]
```diff
@@ -1,110 +0,0 @@
-# Crossplane
```
Comment (csantanapr) [inline modules/crossplane/README.md:1]: Removing crossplane README.md by mistake?

File: modules/kyverno/README.md [hunk 2]
```diff
@@ -1,53 +0,0 @@
-# Kyverno
```
Comment (csantanapr) [inline modules/kyverno/README.md:1]: Why remove Kyverno?

**Comment by csantanapr** [comment]
your going to hate me for asking for this. 🙇 

This PR removes 38 modules, more than half of the total modules.

I'm guessing every single one of them there is a good reason to remove.

I'm thinking that would be good to write down the reason for each module being remove a small sentence on why is being remove, and if there is a better way, alternative way, or new location on how to install the module to provide a link.

This will provide to any user looking for a module they were using, to know why is not present, to NOT open an issue to add the software module back, and a bit of guidance on how to accomplish the desire outcome related to the removed module.

**Comment by bryantbiggs** [comment]
@csantanapr the addons here are a copy of whats in EKS Blueprints. When we remove the addons from EKS Blueprints and encourage users to migrate, we will have documentation on the specifics at that time. For now, these are not in use and we are working on refactoring to suit the needs of users in a scalable fashion

**Comment by csantanapr** [comment]
@bryantbiggs Ah ok this is the new location no one would look here for the old ones, when we remove them from the git repo https://github.com/aws-ia/terraform-aws-eks-blueprints then in that repo will have the list of modules being remove and can have the reason and migration doc for each
👍

File: modules/kyverno/README.md (hunk 2 above)
Comment (Aym3nTN) [reply modules/kyverno/README.md:1]: Why?

**Pull request merged**